### DISCUSSION TITLE
How to solve ''core.symlinks=false'' ''pull.rebase=false''   ''core.filemode=false & core.bare=false :''  problems?

### DISCUSSION THREAD

**Discussion by ANoyanyasadi** [msys2/MSYS2-packages#4866] [Q&A]
Hello, 
I am getting  ''core.symlinks=false'' and ''pull.rebase=false'',   ''core.filemode=false & core.bare=false :  messages on git bash when I type ''git config --list'' in order to create new repo. 

Would you please be so kind to guide me how I can move forward?, also disabling false messages ? I have restarted windows, however got no positive affect. 

Many thanks,
Kind Regards,

**Comment by rimrul** [comment]
> I am getting ''core.symlinks=false'' and ''pull.rebase=false'', ''core.filemode=false & core.bare=false : messages on git bash when I type ''git config --list'' 

Which of these do you consider a problem and why? Which options did you choose when installing Git for Windows?
